go.mod

```diff
@@ -55,3 +55,5 @@ require (
 )
 
 replace go.etcd.io/bbolt => go.etcd.io/bbolt v1.3.5
+
+replace github.com/pingcap/errors => github.com/overvenus/errors v0.11.5-0.20200929094254-677c1712148a
```

Comment:
need update?

Reply:
This PR should wait until https://github.com/pingcap/errors/pull/28 is merged as I mentioned in the PR description